CHANGELOG.asciidoc

```diff
@@ -23,6 +23,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 
 *Metricbeat*
 
+- Fix division by zero on diskio metricset {issue}9124[9124] {pull}9125[9125]
```

Comment:
nit: missing `.` at the end of the sentence.

Reply:
Thanks, fixed